website/content/api-docs/hcp-link.mdx

```diff
@@ -0,0 +1,186 @@
+---
+layout: api
+page_title: HCP Linking HTTP API
+description: The Link resource allows for linking your cluster to HCP Consul Central.
+---
+
+# Link HTTP API
+
+-> **1.18.0+:** The Link API is available in Consul versions 1.18.0 and newer.
+
+## Establish or Update Link to HCP Consul Central
+
+This endpoint creates or updates a Link, which establishes a connection with HCP Consul Central.
+
+| Method | Path                          | Produces           |
+| ------ | ----------------------------- | ------------------ |
+| `PUT`  | `/api/hcp/v2/link/global`     | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/consul/api-docs/features/blocking),
+[consistency modes](/consul/api-docs/features/consistency),
+[agent caching](/consul/api-docs/features/caching), and
+[required ACLs](/consul/api-docs/api-structure#authentication).
+
+| Blocking Queries | Consistency Modes     | Agent Caching | ACL Required                   |
+| ---------------- | --------------------- | ------------- | ------------------------------ |
+| `NO`             | `stale`, `consistent` | `none`        | `operator:write`, `acl:write`  |
+
+### JSON Request Body Schema
+
+- `data` `(object: <required>)`
+
+  - `resourceId` `(string: <required>)` - The ID of the HCP Consul Central cluster to link to. Has the format of:
+    `organization/<orgID>/project/<projectID>/hashicorp.consul.global-network-manager.cluster/<clusterName>`
+
+  - `clientId` `(string: <required>)` - The ID used to authenticate to HCP, which is returned as part of the HCP
+    Consul Central cluster creation. It can also be obtained by fetching the HCP Consul Central cluster secrets.
+
+  - `clientSecret` `(string: <required>)` - The secret used to authenticate to HCP, which is returned as part of the
+    HCP Consul Central cluster creation. It can also be obtained by fetching the HCP Consul Central cluster secrets.
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "resourceId": "organization/c0bf7aac-7690-4905-a8aa-889df1510314/project/6e82a47b-79af-4920-ad42-c0f74421ab52/hashicorp.consul.global-network-manager.cluster/my-cluster",
+    "clientId": "3jz6zk2tlr802htzwquczlxlsrohlpm5",
```

Comment:
The clientId/clientSecret I used here are random strings I generated and pasted in place of the real ones that I used to generate these samples.